db/migrate/20161119134300_add_organizer_to_workshops.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizerToWorkshops < ActiveRecord::Migration
+  def change
+    add_column :workshops, :organizer, :string
```

Comment:
This migration only leads to problems if you migrate rename first and then this migration. But since rename was on dev first, this one should be changed, since everyone who already migrated that will have a problem. (My database works because I resetted and migrated again, but like bjarne said, this should not be necessary)